services/go-importer/cmd/assembler/main.go

```diff
@@ -199,6 +263,17 @@ func handlePcap(fname string) {
 	streamPool := reassembly.NewStreamPool(streamFactory)
 	assembler := reassembly.NewAssembler(streamPool)
 
+	var nextFlush time.Time
+	var flushDuration time.Duration
+	if *flushAfter != "" {
+		flushDuration, err := time.ParseDuration(*flushAfter)
```

Comment:
This redefines the flushDuration variable, which causes it's value to be 0 later on.

```suggestion
	var flushDuration time.Duration
	var err error
	if *flushAfter != "" {
		flushDuration, err = time.ParseDuration(*flushAfter)
```